packages/headless/package.json

```diff
@@ -64,8 +64,14 @@
     "node-abort-controller": "^3.0.0",
     "pino": "8.19.0",
     "redux-thunk": "3.1.0",
+    "remark-gfm": "4.0.0",
```

Comment:
Oof, that's a lot of new dependencies. I'm not a fan of shipping all of this in our package 😕 

That's almost a 50% increase in bundle size, and most of these libs haven't been updated in more than 6 months.

Could we take another approach? Maybe have an endpoint that can convert markdown to sanitized HTML?